decidim-meetings/app/controllers/decidim/meetings/meetings_controller.rb

```diff
@@ -6,9 +6,33 @@ module Meetings
     class MeetingsController < Decidim::Meetings::ApplicationController
       include FilterResource
 
-      helper_method :meetings, :geocoded_meetings, :meeting
+      helper_method :meeting
 
-      def index; end
+      def index
+        results = search.results
+
+        if !params[:filter] && results.length == 0
+          params[:filter] = {
+            date: "past",
+            meetings_warning: true
+          }
+          results = search_klass.new(search_params).results
+          if results.length == 0
+            params[:filter] = {
```

Comment:
Something like:

```ruby
      def index
        if search.results.length == 0 && params.dig("filter", "date") != "past" # Maybe this could be extracted to a method
          @search = search_klass.new(search_params.merge(filter: { date: "past" }))
        end
      end
```